src/Main.java

```diff
@@ -1,5 +1,72 @@
+import java.util.Random;
+import java.util.Scanner;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        // task 1
+        System.out.println(findSymbolOccurance("Java",'a'));
+        // task 2
+        System.out.println(findWordPosition("Apollo", "pollo"));
+        System.out.println(findWordPosition("Apple","Planet"));
+        // task 3
+        System.out.println(stringReverse("Hello"));
+        System.out.println(stringReverse2("Java"));
+        // task 4
+        System.out.println(isPalindrome("ERE"));
+        System.out.println(isPalindrome("Java"));
+        // task 5
+        String[] words = {"apple", "orange", "lemon", "banana", "apricot", "avocado", "broccoli", "carrot", "cherry", "garlic", "grape", "melon",
```

Comment:
Could you please extract the word guessing game into a separate method? This will make your code in main method more readable